src/OrchardCore.Themes/TheTheme/Views/Home/Index.cshtml

```diff
@@ -1,3 +1,3 @@
 <div class="message message-success">
-    <h4>@T["Welcome to the Orchard Framework, your site has been successfully set up."]</h4>
```

Comment:
This is confusing while the Orchard Framework is a set of APIs that could be used in Orchard Core CMS and any other app

Reply:
Welcome to Orchard Core. You can drop the "the".